src/components/date_picker/super_date_picker/super_date_picker.tsx

```diff
@@ -87,6 +87,11 @@ export type EuiSuperDatePickerProps = CommonProps & {
    */
   locale?: LocaleSpecifier;
 
+  /**
+   * Triggered whenever the EuiSuperDatePicker is focused
```

Comment:
```suggestion
   * Triggered whenever the EuiSuperDatePicker's dates are focused
```

Since we're not triggering focus on the quick select button or the refresh button, I think we need to specify this